### DISCUSSION TITLE
Quantification based on Unique peptides

### DISCUSSION THREAD

**Discussion by Maithy15** [vdemichev/DiaNN#251] [General]
Is it possible to make DIA-NN quantify based on unique peptides only? or is this already given by the unique gene matrix in output?

**Comment by vdemichev** [comment]
Genes.MaxLFQ.Unique is this quantity. Also, you can always perform MaxLFQ quant in R yourself using the diann R package or the iq R package, and whatever peptides you'd like to use for this

**Reply by vdemichev** [reply]
Yes, the unique genes matrix is also based on unique peptides only
